include/viint.h

```diff
@@ -6,11 +6,11 @@
 #include "PR/ultratypes.h"
 #include "os.h"
 
-typedef struct {
-    /* 0x0 */ f32 factor;
-    /* 0x4 */ u16 offset;
-    /* 0x8 */ u32 scale;
-} __OSViScale;
+// typedef struct {
+//     /* 0x0 */ f32 factor;
+//     /* 0x4 */ u16 offset;
+//     /* 0x8 */ u32 scale;
+// } __OSViScale;
```

Comment:
Is this struct intended to be commented?

Reply:
I don't know where these OSVi structs are best placed.